unison-src/transcripts/find-patch.md

```diff
@@ -23,6 +23,5 @@ Update
 ```ucm
 .> update
 .> find.patch
-.> view.patch patch
 .> view.patch 1
```

Comment:
This started failing because `view.patch` now overwrites numbered args, but I'm not sure what the original `view.patch patch` was meant to be showing anyways, since it doesn't have anything to do with the behaviour of `find.patch` 🤷🏼‍♂️

Reply:
The only thing I can think of that it could be showing is the existence of a patch named `patch`. I guess you could swap 27 and 26 and get the old result, but this is fine too; IMO the transcripts should do a better job of describing what they're demonstrating and why it's important (I'm surely guilty of not doing this too), otherwise it's unfortunately fair game for refactoring.

Reply:
We've previously discussed _not_ overwriting numbered args, but doing some kind of sub-numbering. e.g. first command produces numbers 1.1–1.n; second command produces 2.1–2.m; or maybe a.1 or 1.a or idk.